src/main/java/com/xiaomi/infra/pegasus/client/ClientOptions.java

```diff
@@ -75,6 +79,8 @@
   private final String authProtocol;
   private final Credential credential;
   private final long sessionResetTimeWindowSecs;
+  private final int maxFetchCount;
```

Comment:
same, you need define detail name to indicate it's only used for `multi_get`

Reply:
ok